webcompat/templates/contributors/sub-nav.html

```diff
@@ -5,59 +5,59 @@
     </a>
   </li>
 
-  <li class="sub-nav-item {% if request.url_rule.endpoint in ['contributors_report_bug', 'contributors_reproduce_bug', 'contributors_diagnose_bug', 'contributors_site_outreach'] %} active {% endif %}">
+  <li class="sub-nav-item {% if 'contributors_bug_' in request.url_rule.endpoint %} active {% endif %}">
     <a class="link-text" href="/contributors/report-bug">
       Working with bugs <span class="nested-sub"></span>
     </a>
     <ol class="nested-sub-nav">
-      <li class="nested-sub-nav-item {% if request.url_rule.endpoint == 'contributors_report_bug' %} sub-nav-active{% endif %}">
+      <li class="nested-sub-nav-item {% if request.url_rule.endpoint == 'contributors_bug_report' %} sub-nav-active{% endif %}">
```

Comment:
We could apply the same logic here and so on.

```python
    <ol class="nested-sub-nav">
      <li class="nested-sub-nav-item {% if request.url_rule.rule == '/contributors/report-bug' %} sub-nav-active{% endif %}">
        <a class="link-text" href="/contributors/report-bug">
          Report a Bug
        </a>
      </li>
```

Reply:
This makes sense.